### PR TITLE
add guidelines run to external plugins with execd

### PR DESCRIPTION
**Pull request by sjwang90**

### CODE REVIEW

File: docs/EXTERNAL_PLUGINS.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ### External Plugins
 
-External plugins are external programs that are built outside of Telegraf that
-can run through an `execd` plugin. These external plugins allow for more flexibility
-compared to internal Telegraf plugins.  
+[External plugins](/EXTERNAL_PLUGINS.md) are external programs that are built outside
```
Comment (ssoroka) [inline docs/EXTERNAL_PLUGINS.md:3]: circular link. :D

Comment (ssoroka) [reply docs/EXTERNAL_PLUGINS.md:3]: oh I see, it's a link to the list of plugins. not what I expected.

File: docs/EXTERNAL_PLUGINS.md [hunk 2]
```diff
@@ -52,6 +54,19 @@ This is a guide to help you set up your plugin to use it with `execd`
...
+  1. Add usage and development instructions in the homepage of your repository for running 
+  your plugin with its respective `execd` plugin. Please refer to 
+  [openvpn](/danielnelson/telegraf-execd-openvpn#usage) and [awsalarms](/vipinvkmenon/awsalarms#installation) 
+  for examples. Include the following steps:
+     1. How to download the release package for your platform or how to clone the binary for your external plugin
+     1. The commands to unpack or build your binary
+     1. Location to edit your `telegraf.conf`
+     1. Configuration to run your external plugin with [inputs.execd](/plugins/inputs/execd), 
+     [processors.execd](/plugins/processors/execd) or [outputs.execd](/plugins/outputs/execd)
+     1. Note if restart of reload of Telegraf is required
```
Comment (ssoroka) [inline docs/EXTERNAL_PLUGINS.md:66]: always required?

**Comment by srebhan** [comment]
Please rebase on master to fix the AppVeyor problem.

**Pull request merged**